rskj-core/src/test/java/org/ethereum/rpc/exception/RskErrorResolverTest.java

```diff
@@ -0,0 +1,128 @@
+package org.ethereum.rpc.exception;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.exc.InvalidFormatException;
+import com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.googlecode.jsonrpc4j.ErrorResolver.JsonError;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by Nazaret García on 26/05/2021
+ */
+
+public class RskErrorResolverTest {
+
+    private RskErrorResolver rskErrorResolver;
+
+    @Before
+    public void setup() {
+        rskErrorResolver = new RskErrorResolver();
+    }
+
+    @Test
+    public void test_resolveError_givenRskJsonRpcRequestException_returnsJsonErrorAsExpected() throws NoSuchMethodException {
+        // Given
```

Comment:
Love the Given / When / Then comments ❤️

Reply:
🙌🏼